data.sql

```diff
@@ -110,12 +110,6 @@ SET character_set_client = utf8mb4 ;
 CREATE TABLE `mirror_log` (`log_id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,`mirror_id` int(10) unsigned NOT NULL DEFAULT '0',`location_id` int(10) unsigned NOT NULL DEFAULT '0',`log_date` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,PRIMARY KEY (`log_id`),KEY `log_date` (`log_date`),KEY `mirror_id` (`mirror_id`),KEY `location_id` (`location_id`)) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 LOCK TABLES `mirror_log` WRITE;
 UNLOCK TABLES;
-DROP TABLE IF EXISTS `mirror_mirrors`;
-SET character_set_client = utf8mb4 ;
-CREATE TABLE `mirror_mirrors` (`id` int(11) NOT NULL AUTO_INCREMENT,`name` varchar(64) NOT NULL DEFAULT '',`baseurl` varchar(255) NOT NULL DEFAULT '',`rating` int(11) NOT NULL DEFAULT '0',`active` tinyint(4) NOT NULL DEFAULT '0',`count` bigint(20) unsigned NOT NULL DEFAULT '0',PRIMARY KEY (`id`),UNIQUE KEY `mirror_name` (`name`),KEY `mirror_count` (`count`)) ENGINE=InnoDB AUTO_INCREMENT=499 DEFAULT CHARSET=utf8;
-LOCK TABLES `mirror_mirrors` WRITE;
-INSERT INTO `mirror_mirrors` VALUES (1,'Mozilla Installer CDN','http://download-installer.cdn.mozilla.net/pub',100000,1,0),(2,'Mozilla Installer CDN - SSL ','https://download-installer.cdn.mozilla.net/pub',81000,1,0);
-UNLOCK TABLES;
 DROP TABLE IF EXISTS `mirror_mirrors_contacts`;
 SET character_set_client = utf8mb4 ;
 CREATE TABLE `mirror_mirrors_contacts` (`id` int(11) NOT NULL AUTO_INCREMENT,`mirror_id` int(11) NOT NULL,`user_id` int(11) NOT NULL,PRIMARY KEY (`id`),UNIQUE KEY `mirror_mirrors_contacts_mirror_id_596bbe5dd7a331f9_uniq` (`mirror_id`,`user_id`),KEY `mirror_mirrors_contacts_mirror_id` (`mirror_id`),KEY `mirror_mirrors_contacts_user_id` (`user_id`)) ENGINE=InnoDB AUTO_INCREMENT=420 DEFAULT CHARSET=latin1;
```

Comment:
shouldn't `mirror_mirrors_contacts` also go?

Reply:
Yes, in the near future most of these tables should go away. This patch unblocks https://bugzilla.mozilla.org/show_bug.cgi?id=1929910, and we can probably  clean up this SQL file in parallel too. I _think_ this SQL file (and another one) are used for localdev/test so that should be fine.